diagnostic.md

```diff
@@ -30,10 +35,10 @@ x = y;
 y = 10;
 ```
 
-After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
-
-<!-- solution below -->
+After each line of code executes, what are the values of x and y?  Do the variables change?  Do their valuolution below -->
 
+x = 3
+y = 10
 
```

Comment:
"Do the variables change?  Do their values?"

The variables do not change, but their values do.